### DISCUSSION TITLE
Welcome to voyager Discussions!

### DISCUSSION THREAD

**Discussion by fletch3555** [thedevdojo/voyager#5157] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lambertn33** [comment]
Hello Everyone i have an issue in Laravel voyager
suppose there is two input types one in input text another is dropdown select..

how could i make input text disabled or not based on dropdown value??

**Comment by austine0608** [comment]
I need help i just uploaded a site but the admin voyager is giving 500 SERVER ERROR can anyone render a solution this is the site url www.lakemfatech.com.ng you can check it out.

Thanks

**Reply by fletch3555** [reply]
As I said in slack, 500 errors are server-side errors.  You'll have to do some basic debugging to identify the specific error.  Once you have the error message and/or stacktrace, you can post it and we'll attempt to help you resolve it.
